perun-registrar-lib/src/test/java/cz/metacentrum/perun/registrar/RegistrarBaseIntegrationTest.java

```diff
@@ -560,6 +564,60 @@ private static void applyForMembershipInVO(RegistrarManager registrarManager, Pe
 		registrarManager.createApplication(user, application, data);
 	}
 
+	@Test
+	public void testEmbeddedGroupsSubmission() throws PerunException {
+		GroupsManager groupsManager = perun.getGroupsManager();
+		ApplicationForm form = registrarManager.getFormForVo(vo);
+		form.setAutomaticApproval(false);
+		form.setAutomaticApprovalEmbedded(false);
```

Comment:
I think this is not actually doing anything, since these changes are not saved to the DB.

Reply:
removed